### ISSUE TITLE
Puzzle display aren't in accordance to game played at times.

### ISSUE THREAD

**Issue by abdul-qudus** [julesbond007/android-jigsaw-puzzle#22]
Expected behavior
When puzzle is been selected, then while playing or making a move. Design or drawings from puzzle should still remain intact.

Actual behavior
When puzzle are relocated, respective drawings keeps changing and sometimes making puzzle uneasy to get.

How to reproduce
Download application.
Draw preferred puzzle of your choice, the select the puzzle option at the top bar. 
Select stage or level to be played. Then after making movement. Responds keep changing at times.

Device: Tecno wx3pro
App version: 1.0
Operating system: Android Nougat 7.0

Recording of the bug
https://youtu.be/kieS6dPgenQ

**Comment by julesbond007** [comment]
@abdul-qudus Yeah it's built into it...you have to keep moving the pieces until it's solved...there is a way to solve it.
